website/models.py

```
@@ -3511,3 +3526,66 @@ class Meta:
+        """
+        Automatically set or clear resolved_at timestamp based on status.
+        """
+        if self.status == self.Status.RESOLVED and not self.resolved_at:
+            self.resolved_at = timezone.now()
+        elif self.status != self.Status.RESOLVED and self.resolved_at:
+            self.resolved_at = None
+
+        super().save(*args, **kwargs)
+
+    class Meta:
+        ordering = ["-created_at"]
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)